android/sdl_android/src/main/java/com/smartdevicelink/util/SdlAppInfo.java

```diff
@@ -228,9 +233,9 @@ public String toString() {
                 eventType = parser.next();
             }
         } catch (XmlPullParserException e) {
-            e.printStackTrace();
+            DebugTool.logError(TAG, "Failed to parse xml file", e);
         } catch (IOException e) {
-            e.printStackTrace();
+            DebugTool.logError(TAG, "Failed to parse xml file", e);
```

Comment:
@YaroslavLutsenko I believe log messages in two catch branches should be different (ex. "Failed to parse XML file" and "General exception was thrown).
BTW isn't `IOException` a base class for `XmlPullParserException`? :)

Reply:
@AKalinich-Luxoft Fixed in the last commit. Both of these exceptions are extended Exception